src/components/HeaderNavMenu.jsx

```diff
@@ -30,8 +52,27 @@ function HeaderNavMenu() {
           </MuiLink>
         </Grid>
       ))}
-      <Grid item>
-        <LoginButton className={classes.menuItem} />
+      <Grid item className={classes.avatarMenu}>
+        {(loading && <null />) ||
```

Comment:
`<null />` probably results in console warning/error messages. You can achieve the same by doing this instead:
```suggestion
        {!loading && 
```

Reply:
Ah ok nice